samcli/commands/local/lib/local_lambda.py

```diff
@@ -43,6 +43,7 @@ def __init__(
         aws_region: Optional[str] = None,
         env_vars_values: Optional[Dict[Any, Any]] = None,
         debug_context: Optional[DebugContext] = None,
+        container_host: Optional[str] = None,
```

Comment:
Since there is a default value for the parameter, would this param be `None` anytime?

Reply:
This param won't be `None` in the current use case. The reasons I set it to `None`:
- I'd like to have minimum impact on the current use case. If this class is initialized somewhere else, we shouldn't assume it's `localhost` unless it's passed from cli.
- Match to the overall style.